csm/cli/schema/auditlog.json

```diff
@@ -0,0 +1,223 @@
+{
+    "name": "auditlog",
+    "description": "Auditlogs",
+    "sub_commands": [
+      {
+        "name": "show",
+        "description": "Display Audit Logs",
+        "permissions_tag": "list",
+        "sub_commands": [
+          {
+            "name": "csm",
+            "description": "Displays CSM Audit Logs on the CLI",
+            "need_confirmation": false,
+            "permissions_tag": "list",
+            "args": [
+              {
+                "flag": "-b",
+                "dest": "start_date",
+                "type": "Validatiors.positive_int",
+                "help": "Start epoch time from which audit logs to be fetched.",
```

Comment:
**Just a thought -** 
If user has to specify an epoch time as input it would require him/her to fetch those values from some sources and then execute the command. 
Instead of epoch can we use date in a simple (yyyy-mm-dd) format so that it will be easier to use the command. 
CC: @kupranay @soniyamoholkar

Reply:
Done. Added a converter for yyyy-mm-dd to epoch time.